metaflow/runtime.py

```diff
@@ -740,10 +740,18 @@ def __init__(self, task):
 
     def get_args(self):
         def options(mapping):
+            """ Map a dictionary of options to a list of command-line arguments.
+
+            Values assigned to boolean values are assumed boolean flags. List values
```

Comment:
Are we assuming boolean flags will be defaulted to false?

Wouldn't it be better to explicitly set the key (k) explicitly based on the value? Thoughts?

Reply:
> Are we assuming boolean flags will be defaulted to false?

That bit of logic is not new, I _think_ that was the intent behind `if value:` in the previous version (but I may be wrong).

Good point on `dict_to_cli_options`, I'll do that

Reply:
You're right - the previous code assumed the same.

I was just pointing out since you were touching this area of code :) no worries if you want to preserve the same semantics for now.